server/src/main/java/io/pravega/schemaregistry/service/Config.java

```diff
@@ -219,7 +219,7 @@ private static ServiceConfig createServiceConfig() {
                                    .tlsEnabled(Config.TLS_ENABLED)
                                    .tlsCertFilePath(Config.TLS_CERT_FILE)
                                    .serverKeyStoreFilePath(Config.TLS_KEY_FILE)
-                                   .serverKeyStoreFilePath(Config.TLS_KEY_PASSWORD_FILE)
+                                   .tlsKeyStorePasswordFilePath(Config.TLS_KEY_PASSWORD_FILE)
```

Comment:
Please make these methods' name similar, as they are related: 

`tlsCertFilePath(..)`,  `tlsKeyStoreFilePath(..)` and `tlsKeyStorePasswordFilePath(..)`

or

`serverCertFilePath(..)`, `serverKeyStoreFilePath(..)` and `serverKeyStorePasswordFilePath(..)`